test/cuda/test_execution_engine_cache.cc

```diff
@@ -66,18 +65,20 @@ def matmul(float(M,K) A, float(K,N) B) -> (output) {
   at::Tensor a2 = at::CUDA(at::kFloat).rand({4, 8});
   at::Tensor b2 = at::CUDA(at::kFloat).rand({8, 7});
   inputs = {a2, b2};
-  std::vector<at::Tensor> outputs2;
-  handle = atCompl.compile("matmul", inputs, mappingOptions);
-  atCompl.run("matmul", inputs, outputs2, handle);
+  auto outputs2 = tc::aten::prepareOutputs(tc, "matmul", inputs);
+  pExecutor =
+      tc::aten::compile<tc::CudaBackend>(tc, "matmul", inputs, mappingOptions);
+  tc::aten::run(*pExecutor, inputs, outputs2);
   diff = outputs2[0].sub(a2.mm(b2));
   checkRtol(diff, inputs, 8); // reduction size is dimension of n
 
   // run the first cached matmul again
```

Comment:
Does this file test anything useful?  It appears to only compile the same thing multiple times, but it never checks that some cache was hit or not.  I'm not even sure that compilation cache exists after refactorings.

Reply:
exactly since there are no more caches ever since before refactoring